### PR TITLE
Update rate limiting documentation.

### PR DESCRIPTION
**Pull request by ShalkiWenushika**
### Purpose
This PR will add some corrections to the local rate limiting documentation.

### Issues

Fixes https://github.com/wso2/docs-mg/issues/94 
Fixes https://github.com/wso2/docs-mg/issues/78

---
#### Maintainers: Check before merge
- [ ] Manual verification

### CODE REVIEW

File: en/docs/concepts/rate-limiting.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+The WSO2 API Microgateway by default has an inbuilt mechanism to handle rate limiting. When creating an API, users can define subscription-level throttling policies. When a Microgateway project gets initialized, an execution plan related to the subscription-level throttling policy gets created in memory. The API Microgateway monitors the incoming requests and publishes the events to a stream defined in the inbuilt stream processing engine. This stream is then read and fed into the relevant pre-defined execution plan. The execution plan detects the throttling point and publishes an event to a stream (globalThrottleStream). When the globalThrottleStream is updated, the API Microgateway updates its local counters in the background as the gateway is subscribed to the global throttle stream.
```
Comment (Rajith90) [inline en/docs/concepts/rate-limiting.md:3]: This content is invalid. This is not how local throttling works at the moment.
And also talking about local throttling here and again mentioning local and distributing throttling below is confusing. Talk generally about rate limiting here and explain each in the below sub topics

File: en/docs/concepts/rate-limiting.md [hunk 2]
```diff
@@ -18,12 +25,12 @@ Based on the selected tier, a subscriber will be throttled out upon reaching the
 
 #### Application-level throttling (application developer)
 
-Application-level throttling tiers are defined at the time an application is created in the API Developer Portal as shown below. The limits are restricted per token for a specific application.
+Application-level throttling tiers are defined at the time an application is created in the API Developer Portal. The limits are restricted per token for a specific application.
```
Comment (Rajith90) [inline en/docs/concepts/rate-limiting.md:28]: Point to the apim application throttling page

File: en/docs/how-tos/rate-limiting/adding-throttling-policies.md [hunk 3]
```diff
@@ -1,12 +1,20 @@
 # Adding Throttling Policies
```
Comment (Rajith90) [inline en/docs/how-tos/rate-limiting/adding-throttling-policies.md:1]: This is only valid for local throttling. Use heading `Define throttle policies for local throttling`

File: en/docs/concepts/rate-limiting.md (hunk 1 above)
Comment (ShalkiWenushika) [reply en/docs/concepts/rate-limiting.md:3]: `The API Microgateway monitors the incoming requests and publishes the events to a stream defined in the inbuilt stream processing engine. This stream is then read and fed into the relevant pre-defined execution plan. The execution plan detects the throttling point and publishes an event to a stream (globalThrottleStream). When the globalThrottleStream is updated, the API Microgateway updates its local counters in the background as the gateway is subscribed to the global throttle stream.` Is this content also invalid?

File: en/docs/concepts/rate-limiting.md [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
 
 The WSO2 API Microgateway supports [resource level]({{base_path}}/how-tos/rate-limiting/adding-throttling-policies/), [subscription level](#subscription-level-throttling-api-subscriber) and [application level throttling](#application-level-throttling-application-developer).
 
-In a lockdown environment or in offline mode where there is no connection with the central traffic management solution, the default node-level throttling can be used.
+In a lockdown environment or in offline mode where there is no connection with the central traffic management solution, the default node-level throttling(local throttling) can be used.
+
+#### Local rate limiting
+If a user wants to rate limit requests within a microgateway, the local rate limiting option can be used. When local rate limiting is enabled, WSO2 API Microgateway maintains a local counter within the microgateway and check whether the request count exceeds the throttling limit. Find information on how to enable local rate limiting from [here](https://mg.docs.wso2.com/en/latest/how-tos/rate-limiting/adding-throttling-policies/).
 
 ![rate limiting-overview]({{base_path}}/assets/img/how-tos/rate-limiting-overview.jpg)
```
Comment (Rajith90) [inline en/docs/concepts/rate-limiting.md:17]: Lets remove the image since it is not valid anymore

File: en/mkdocs.yml [hunk 5]
```diff
@@ -89,8 +89,8 @@ nav:
         - API Schema Validation: how-tos/api-schema-validation.md
         - HTTP 2.0: how-tos/http-2.0.md
         - Rate Limiting:
-            - Distributed Throttling: how-tos/rate-limiting/distributed-throttling.md
-            - Adding Throttling Policies: how-tos/rate-limiting/adding-throttling-policies.md
+            - Enabling distributed throttling: how-tos/rate-limiting/distributed-throttling.md
+            - Enabling local throttling: how-tos/rate-limiting/adding-throttling-policies.md
```
Comment (Rajith90) [inline en/mkdocs.yml:93]: We need to add redirects to this links. otherwise old links will not work

Comment (ShalkiWenushika) [reply en/mkdocs.yml:93]: Links are same as the previous. I only changed the Title of the document.

**Review by Rajith90 — APPROVED** [review]

**Pull request merged**